Src/CSharpier/Printer/ArgumentListSyntax.cs

```diff
@@ -7,25 +7,39 @@ public partial class Printer
     {
         private Doc PrintArgumentListSyntax(ArgumentListSyntax node)
         {
-            var result = Concat(
-                this.PrintSyntaxToken(node.OpenParenToken),
-                node.Arguments.Any()
+            return node.Parent is not ObjectCreationExpressionSyntax
+                ? Group(
+                    PrintArgumentListLikeSyntax(
+                        node.OpenParenToken,
+                        node.Arguments,
+                        node.CloseParenToken
+                    )
+                )
+                : PrintArgumentListLikeSyntax(
+                    node.OpenParenToken,
+                    node.Arguments,
+                    node.CloseParenToken
+                );
+        }
+
+        private Doc PrintArgumentListLikeSyntax(
```

Comment:
Not sure where you'd want to extract shared code between syntax tree expressions. I put it here because it's the most related thing I could think of, but feel free to suggest elsewhere.

Reply:
Right now it is a mix between methods in PrinterHelpers and methods directly in the Printer.cs class.
I think I lean towards keeping them in PrinterHelper files. At some point PrinterHelpers didn't exist, but the Printer.cs file was getting too big so I pulled things out as I worked on them.

Reply:
Okay, I moved it to `PrinterHelpers/`. We should find a naming convention that lets us easily distinguish the methods that handle a Roslyn syntax node vs these helpers.

Personally, I'm not a big fan of the term "helper", because it's such a generic term that doesn't really mean anything. But I don't have any real suggestions yet, so I'll go with it for now.